client/src/main/java/envoy/client/ui/settings/BugReportPane.java

```diff
@@ -0,0 +1,76 @@
+package envoy.client.ui.settings;
+
+import javafx.event.EventHandler;
+import javafx.scene.control.*;
+import javafx.scene.input.InputEvent;
+
+import envoy.client.event.SendEvent;
+import envoy.data.User;
+import envoy.event.EventBus;
+import envoy.event.IssueProposal;
+
+/**
+ * This class offers the option for users to submit a bug report. Only the title
+ * of a bug is needed to be sent.
+ * <p>
+ * Project: <strong>client</strong><br>
+ * File: <strong>BugReportPane.java</strong><br>
+ * Created: <strong>Aug 4, 2020</strong><br>
```

Comment:
If I remember correctly, there is no `<br>` tag in this line. However, I think this header is not very useful anyway, so we might as well replace it by something more concise.

```suggestion
 * Created: <strong>Aug 4, 2020</strong>
```

Reply:
![image](https://user-images.githubusercontent.com/51889757/90760226-d78bbe00-e2e1-11ea-98c8-ff1fd7f11622.png)